profiles/common/modules/features/node_pager/node_pager.install

```diff
@@ -20,22 +20,7 @@ function node_pager_enable() {
  * Implements hook_disable().
  */
 function node_pager_disable() {
-  //Example
-  //multisite_drupal_toolbox_disable_content_type('kit');
-
+  // Example
+  // multisite_drupal_toolbox_disable_content_type('kit');.
```

Comment:
Hmm this documentation seems to be incorrect, there is just a message being shown to inform the user that the Node Pager is disabled. I would just remove this documentation.